devicetypes/smartthings/zigbee-lock.src/zigbee-lock.groovy

```diff
@@ -28,6 +28,8 @@ metadata {
 		capability "Health Check"
 
 		fingerprint profileId: "0104", inClusters: "0000,0001,0009,000A,0101,0020", outClusters: "000A,0019", manufacturer: "Yale", model: "YRD220/240 TSDB", deviceJoinName: "Yale Door Lock" //Yale Touch Screen Deadbolt Lock
+		fingerprint profileId: "0104", inClusters: "0000,0001,0003,0004,0005,0009,000A,0101", outClusters: "0019", manufacturer: "TOTEM", model: "H60/H90", deviceJoinName: "ECO Door Lock" //TOTEM Door lock
+		fingerprint profileId: "0104", inClusters: "0000,0001,0003,0004,0005,0009,000A,0101", outClusters: "0019", manufacturer: "TOTEM", model: "P30", deviceJoinName: "ECO Door Lock" //TOTEM Door lock
```

Comment:
Since the manufacturer is `TOTEM` please change  deviceJoinNames to  `"TOTEM Door Lock"`

Reply:
@LUZhanchang